packages/ruleset/test/test-utils/root-document.js

```diff
@@ -979,11 +983,15 @@ module.exports = {
               'The number of items returned in this page of results.',
             type: 'integer',
             format: 'int32',
+            minimum: 0,
```

Comment:
Limits should be positive integers

```suggestion
            minimum: 1,
```

Reply:
fixed